jdaviz/configs/specviz/plugins/line_analysis/line_analysis.py

```diff
@@ -97,6 +100,15 @@ def __init__(self, *args, **kwargs):
         # require entries to be in spectrum-viewer (not other cubeviz images, etc)
         self.dataset.add_filter('layer_in_spectrum_viewer')
 
+        if self.app.state.settings.get("configuration") == "cubeviz":
+            self.spatial_subset = SubsetSelect(self,
+                                               'spatial_subset_items',
+                                               'collapsed_spectrum_selected',
+                                               default_text='Entire Cube Spectrum',
+                                               allowed_type='spatial')
```

Comment:
I'd advocate for using the same traitlet names and default text as in model fitting so there is a consistent UX and API for the user between the two.  If there are subtle differences that we want to capture in the UI, then the hint might be the easiest place to make that distinction.  (Or we can always change model fitting to match if there is a good reason to make changes).

Reply:
I'd still like to see this addressed/discussed before approving/merging!

Reply:
I was going to address this in my "review phase" now that the CI is passing. I think I've mellowed on my original intent, so I'm happy to switch it back! Let me do that first and I'll rerequest review

Reply:
I am not familiar with model fitting, so maybe @rosteen or @javerbukh would have opinions.

Reply:
Effectively the distinction I tried to make is between selecting the spatial subset itself, or the spectrum generated from the spatial subset. Effectively, it looks identical on the front end, but it's slightly different here in the backend

Reply:
@kecnry I've reverted those changes so they should match; let me know if it's sufficient, or if I missed something!

Reply:
I do see your point, but just really want to strive for self-consistency across plugins, especially with attribute names and options since we're looking to expose these directly in #1401.  This still differs from model fittings spatial dropdown (which is the same use-case: acting on the spectrum corresponding to the referenced spatial subset), in that model fitting has "Entire Cube" and this has "Entire Cube Spectrum".  I definitely think they should be the same, and personally would vote for "Entire Cube" and add any clarification to the user in the dropdown hint and/or docs.  But if others prefer being more verbose, then let's also make the change to model fitting.

Reply:
Ok, I changed the default text/labeling as well. I reworked the `hint` to try and make it explicit. How does that look @kecnry ?